extensions/test-extensions.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// Extensions used for local development and tests. These changes will not be
+// included in the built dist.
+
+// This is required for our integration tests, which tests whitespaced
+// namespaces for Create from URL.
+window.OPENSHIFT_CONSTANTS.CREATE_FROM_URL_WHITELIST = [
```

Comment:
so are we always running with this extension in place in dev mode then? i don't love creating this pattern, what if someone breaks the OOTB constants and doesn't realize it because the extension constants are overriding it in dev mode?

Reply:
i think this one would bother me less if we pushed template-dumpster onto the array, instead of overwriting the whole array

Reply:
I was considering suggesting we keep the `/helpers/extensions.js` and update the `addExtension()` fn to read `index.html` && write directly to the `head` of the document only when the test runs.